packages/dev/docs/src/docs.css

```diff
@@ -72,7 +72,16 @@ html, body {
 }
 
 .dark {
-  --page-background: var(--spectrum-global-color-gray-100);
+  --page-background: var(--spectrum-global-color-gray-75);
+  nav {
+    background-color: var(--spectrum-global-color-gray-100);
```

Comment:
Some people have used background then color and others used background-color. I preferred the specificity.